src/test/udp/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+find_package(GLIB REQUIRED)
+include_directories(${GLIB_INCLUDES})
+link_libraries(${GLIB_LIBRARIES})
+
+## build the test as a dynamic executable that plugs into shadow
+add_shadow_plugin(shadow-plugin-test-udp shd-test-udp.c)
+
+## create and install an executable that can run outside of shadow
+add_executable(test-udp shd-test-udp.c)
```

Comment:
Can/should we just run the `test-udp` binary in Shadow too and that way we don't have to build `libshadow-plugin-test-udp.so` too?

I think we can do that like:

```diff
- ## build the test as a dynamic executable that plugs into shadow
- add_shadow_plugin(shadow-plugin-test-udp shd-test-udp.c)
-
- ## create and install an executable that can run outside of shadow
- add_executable(test-udp shd-test-udp.c)
+ ## build a position-independent executable that can run inside or outside of shadow
+ add_shadow_exe(test-udp shd-test-udp.c)
```

Reply:
Done